metpy/calc/tools.py

```diff
@@ -1126,3 +1135,52 @@ def _process_deriv_args(f, kwargs):
         raise ValueError('Must specify either "x" or "delta" for value positions.')
 
     return n, axis, delta
+
+
+@exporter.export
+def parse_angle(input_dir, deg2rad=False):
```

Comment:
I'm wondering about not having the extra kwarg of `deg2rad` and just return degrees by default. Thanks to unit handling, if other calculations need radians, they'll handle the conversion. If a user needs it, its easy to convert.